Modules/signalmodule.c

```diff
@@ -1792,11 +1792,42 @@ PyErr_CheckSignals(void)
     return _PyErr_CheckSignalsTstate(tstate);
 }
 
+#if defined(__EMSCRIPTEN__)
+
+#include <emscripten.h>
+EM_JS(int, _Py_CheckEmscriptenSignals_Helper, (void), {
+    if(!Module.Py_EmscriptenSignalBuffer){
+        return 0;
+    }
+    let result = Module.Py_EmscriptenSignalBuffer[0];
+    Module.Py_EmscriptenSignalBuffer[0] = 0;
+    return result;
+});
+
+void
+_Py_CheckEmscriptenSignals(void)
+{
+    int signal = _Py_CheckEmscriptenSignals_Helper();
+    if (signal) {
+        PyErr_SetInterruptEx(signal);
+    }
+}
+
+
+int Py_EMSCRIPTEN_SIGNAL_HANDLING = 0;
```

Comment:
How is this variable set? I don't see any code that changes the global value to ``1``. Does Pyodide modify the value through WASM globals / exports? Please add a comment that explains how 3rd party code should use the flag.